public/style.css

```
@@ -20,7 +20,8 @@
 @import url("native/session/session-info.css");
 @import url("native/workspace/header-open-app.css");
 @import url("native/extensions/command-palette.css");
-@import url("native/features/lan-qr.css");
+@import url("native/features/app-launcher.css");
```

Stylelint (17.14.0)

[error] Expected "url("native/features/app-launcher.css")" to be ""native/features/app-launcher.css"" (import-notation)

(import-notation)